### ISSUE TITLE
Add network (net) parameter to docker template settings

### ISSUE THREAD

**Issue by michelvocks** [KostyaSha/yet-another-docker-plugin#24]
Hello all,

I would really like to see the "--net" parameter in the docker template settings.
That allows me to start my jenkins slave in a predefined network.
Or is there any work around that I could use to set this parameter?

Greetings

**Comment by KostyaSha** [comment]
Would look. Could you describe what exactly should it affect? Only interface name?

**Comment by KostyaSha** [comment]
in docker-java i see
```
    /**
     * Set the Network mode for the container
     * <ul>
     * <li>'bridge': creates a new network stack for the container on the docker bridge</li>
     * <li>'none': no networking for this container</li>
     * <li>'container:<name|id>': reuses another container network stack</li>
     * <li>'host': use the host network stack inside the container. Note: the host mode gives the container full access
     * to local system services such as D-bus and is therefore considered insecure.</li>
     * </ul>
     */
```
Is it what you want?

**Comment by KostyaSha** [comment]
#25 need to be tested, searching the way to share test hpi.

**Comment by michelvocks** [comment]
Yes, this is exactly what I need. Would love to test it.

**Comment by KostyaSha** [comment]
@michelvocks Please try this hpi https://www.dropbox.com/s/j19xsw9ibwg2l1f/yet-another-docker-plugin.hpi?dl=0

**Comment by KostyaSha** [comment]
@michelvocks does it work for you?

**Comment by KostyaSha** [comment]
@michelvocks ping?

**Comment by neonman63** [comment]
It works for me

**Comment by KostyaSha** [comment]
cool, thanks

**Comment by KostyaSha** [comment]
Released as 0.1.0-rc8 (yeah versioning sucks).
